docs/n3/foundation/Transactions.md

```diff
@@ -168,3 +154,258 @@ Here is an example of a JSON-format transaction, where the script and witnesses
 }
 ```
 
+## Signature Scope
+
+In Neo Legacy transaction signature is globally effective. In order to allow users to control the signature scope at a finer level of granularity, WitnessScope is added to  Neo N3 and the signers field in the transaction structure is changed, so that the signature can be used only for verifying the specified contract, preventing unauthorized contracts from using the user signature.
+
+### Scopes
+
+When constructing a transaction, you need to specify the field `scopes` in `signers`, which defines the effective range of the signature, including these types:
+
+| **Field** | **Name**          | **Description**                                              |
+| --------- | ----------------- | ------------------------------------------------------------ |
+| 0x00      | `None`            | The signature is used for transactions only, and is disabled in contracts |
+| 0x01      | `CalledByEntry`   | The signature is only effective to the contract script called by Entry. |
+| 0x10      | `CustomContracts` | The signature is only effective to the specified contract script. It can be used in conjunction with CalledByEntry. |
+| 0x20      | `CustomGroups`    | The signature is effective to contracts in the group. It can be used in conjunction with CalledByEntry. |
+| 0x80      | `Global`          | The signature is globally effective. The risk is extremely high because the contract may transfer all assets in the address. Only choose it when the contract is extremely trusted. |
+| 0x40      | `WitnessRules`    | You need to specify the rule and scope. See [WitnessRule](#witnessrule) |
+
+For better understanding, suppose there is a contract invocation chain: **[entry]->[Contract A]->[Contract B]->[Contract C]...->[Target]**
+
+And the Target contract invokes CheckWitness to verify the signature. The verification result varies when `scopes` is set to different value. 
+
+- `None` - The verification is passed no matter where the **Target** contract is.
+- `Global` - The verification is passed no matter where the **Target** contract is.
+- `CallByEntry` - The verification is passed only when the **Target** contract is **entry** or **Contract A**.
+- `CustomContracts` - The verification is passed only when the **Target** contract belongs to **CustomContracts**, a contract list you need to customize.
+- `CustomGroups` - The verification is passed only when the **Target** contract is authenticated by any public key in **CustomGroups**, a public key group you need to customize.
+
+### WitnessRule
+
+Action(Allow|Deny) and Condition
+
+The execution logic is to execute the condition and, if is met, return Action, where Allow represents a successful check and Deny represents a failed check.
+
+#### WitnessCondition
+
+- Boolean: true|false
+
+  “expression” = <bool>
```

Comment:
I think the easiest way is to find and replace in this doc. 

Find: < and Replace: \<
Find: > and Replace: \>

Reply:
I have done the replacement. Please kindly check.